packages/ui/src/atom/input-atom/input-atom.vue

```diff
@@ -2,11 +2,12 @@
   <span
     class="input-atom"
     :class="inputAtomCssClass"
-    :disabled="$attrs.disabled"
+    :disabled="disabled"
```

Comment:
`disabled` on `span`? Why?

Reply:
I guess, some styles depends on this attribute
@dudiq am I right?

Reply:
there are two ways:
- use `IsDisabledMixin` for get correct `disabled` value and make class names for elements. mixin needed because you can pass `disabled` as attribute and value will be `undefined`. it will be semantic, but with `using mixin`
- stay as is with using disabled attr. not semantic but without mixin usage and more logic.

@kazurus said that `stay as is without mixin usage`

Reply:
Actually you don't need any mixin, because if you just want to add `class` you need only know the fact that attribute or prop was passed. Any mixins are useless. You don't need to format incoming value, only except if you write something kinda `disabled="false"`, but this is incorrect usage.

Anyway, now there is opportunity to change attribute-based styles to class-based ones. @kazurus should I do that? Or leave it as is?

Reply:
not now